.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+#
+# Release workflow for Mount Helper Script.
+#
+
+name: Release to github for Mount Helper
+
+on:
+  workflow_dispatch:
+
+
+jobs:
+  package_mount_helper:
+    name: Package and Release Mount Helper
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+    - name: Run Package.sh
+      run: |
+        chmod +x ./package.sh
+        chmod +x ./packaging/aznfs/DEBIAN/*
+        ./package.sh
+      shell: bash
+    - name: Create Release
+      id: create_release
+      uses: actions/create_release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
+        release_name: Release ${{ github.ref }}
+        body: |
+          # This could be filled depends on the release.ref
+        draft: false
+        prerelease: false
+    - name: Upload Release Asset
+      id: upload-release-asset
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ steps.crete_release.outputs.upload_url }}
+        asset_path: ./aznfs.deb
+        asset_name: aznfs.deb
```

Comment:
does it automatically have the release/build number added?
If not, we need to ensure that the artifacts/assets generated are properly tagged

Reply:
I have to test this by running the pipeline in github. that I can only do after this change is merged into default branch of the repo (main branch).